theme/templates/article.html

```diff
@@ -68,5 +68,9 @@ <h4>{% trans %}Comments{% endtrans %}</h4>
 {% block extra_head %}
 {% if article.comments and article.comments in ("true", "yes") %}
 <script src="{{ SITEURL }}/theme/js/discourse.js"></script>
+{% if article.math and article.math in ("true", "yes") %}
+{# Note that we don't insert any mathjax config. If you want to configure mathjax, see https://docs.mathjax.org/en/latest/web/configuration.html#configuring-mathjax #}
+<script id="MathJax-script" type="text/javascript" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js"></script>
+{% endif %}
```

Comment:
I slightly misplaced this block (its not obvious because there is no indentation). Just move the `{% endif %}` to the top. This may be why mathjax is broken for you?

```suggestion
{% endif %}
{% if article.math and article.math in ("true", "yes") %}
{# Note that we don't insert any mathjax config. If you want to configure mathjax, see https://docs.mathjax.org/en/latest/web/configuration.html#configuring-mathjax #}
<script id="MathJax-script" type="text/javascript" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js"></script>
```

Reply:
I cherry-picked the latest commit from you PR again and tried your suggested change, but to no avail.

Reply:
yeah, I can reproduce again now. I'll try to find some time to look into it. Probably not today though.

Reply:
Please take another look at the other PR again. I added some subresource integrity stuff that should make this work.

Reply:
Should be up-to-date now